chromium_src/net/http/transport_security_state.cc

```diff
@@ -0,0 +1,54 @@
+/* Copyright 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "net/http/transport_security_state.h"
+
+#include <algorithm>
```

Comment:
what are all these includes for?

Reply:
I see https://github.com/brave/brave-core/pull/8750/files#r629528206 but GOOGLE_CHROME_BRANDING would never apply to most of these and certainly not to system includes.

Reply:
It's usually a pretty fragile method for overriding that we avoid if possible, but less of an issue outside of src/chrome because GOOGLE_CHROME_BRANDING generally does not appear in headers in other directories. This looks like an upstream bug that should be `IS_OFFICIAL_BUILD`. @mkarolin I think we want to avoid using this method of overriding unless it's a case where we'd have to do significant patching. The risk of unintended changes is too high. @fmarier I would patch in BRAVE_TRANSPORT_SECURITY_STATE right after `enable_static_expect_ct_ = false;` and set them both back to true

Reply:
> an upstream bug that should be `IS_OFFICIAL_BUILD`

The list of pinned hosts is actually intended to be Google-only. Google has asked derivatives not to enable their pins so that mistakes in their list don't make it to downstream users.

> I would patch in BRAVE_TRANSPORT_SECURITY_STATE right after enable_static_expect_ct_ = false; and set them both back to true

Ok, sounds good. I will replace this with a small targeted patch.

Reply:
Fixed in latest revision.